### PR TITLE
Correct behaviour for Android >= 6

### PR DESCRIPTION
**Pull request by AntonioVdlC**
Yeah ... so basically the code I removed because I thought it was useless ... well actually it broke the permissions for Android >= 6 😬 

I'm also removing the permission popup that appears on launch as the GPS web whisper should take over (but I'm leaving it in comments just in case)!

### CODE REVIEW

**Comment by AntonioVdlC** [comment]
Waiting on this to be close before closing https://github.com/cabify/cabify_coffee/pull/1339!

File: src/android/CordovaGPSLocation.java [hunk 1]
```diff
@@ -79,6 +79,11 @@ public boolean execute(final String action, final JSONArray args,
             return false;
         }
 
+        if(!hasPermisssion()) {
+            PermissionHelper.requestPermissions(this, 0, permissions);
+            return true;
```
Comment (spinillos) [inline src/android/CordovaGPSLocation.java:84]: Why returns true when no permission?

Comment (AntonioVdlC) [reply src/android/CordovaGPSLocation.java:84]: I honestly don't know, that's how it was ... 😜

**Comment by AntonioVdlC** [comment]
Merging it after @lurecas review!

**Pull request merged**